crates/scripts/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use std::collections::HashMap;
+
+use ethers::abi::Token;
+use kakarot_rpc_core::test_utils::deploy_helpers::{
+    ContractDeploymentArgs, KakarotTestEnvironmentContext, TestContext,
+};
+use katana_core::db::Db;
+
+#[tokio::main]
+async fn main() {
+    // Deploy all kakarot contracts + EVM contracts
+    let mut test_context = KakarotTestEnvironmentContext::new(TestContext::PlainOpcodes).await;
+    test_context = test_context
+        .deploy_evm_contract(ContractDeploymentArgs {
+            name: "ERC20".into(),
+            constructor_args: (
+                Token::String("Test".into()),               // name
+                Token::String("TT".into()),                 // symbol
+                Token::Uint(ethers::types::U256::from(18)), // decimals
+            ),
+        })
+        .await;
+
+    tokio::task::spawn_blocking(move || {
+        // Get a serializable state for the sequencer
+        let sequencer = test_context.sequencer();
+        let dump_state =
+            sequencer.sequencer.backend.state.blocking_write().dump_state().expect("Failed to dump Katana state");
```

Comment:
We should probably make the expect failure messages more accurate in order to properly catch the reason for the failure
since every expect is the same it'll be hard to debug down the line